index.js

```diff
@@ -32,7 +32,7 @@ var StyleDictionary = {
   transform: require('./lib/common/transforms'),
   transformGroup: require('./lib/common/transformGroups'),
   format: require('./lib/common/formats'),
-  template: require('./lib/common/templates'),
+  // template: require('./lib/common/templates'),
```

Comment:
remove it?

Reply:
Good catch! I had commented during testing but definitely don't want to include in code.